text/3771-demote-i686-pc-windows-gnu.md

```diff
@@ -0,0 +1,172 @@
+
+Some example problems, found by searching for `ignore-windows-gnu` in rust-lang/rust.
+
+- https://github.com/rust-lang/rust/issues/128973
+- https://github.com/rust-lang/rust/issues/128981
+- https://github.com/rust-lang/rust/pull/116837
+- https://github.com/rust-lang/rust/issues/128911
+
+### 32-bit x86 Problems
+
+While some of these issues apply to all GNU-based targets, 32-bit x86 seems to be especially affected.
+And when a 32-bit Windows GNU issue comes up, contributors rarely actually investigate it, because it is such a complex and nonstandard environment compared to 64-bit Windows GNU, which is a lot easier to set up and work with.
+
+That the 32-bit x86 architecture is unusual, and made moreso by how Windows operates on it, has also been noted by Windows experts[^2].
+The Windows GNU experts that provide direct support to the Rust project focus almost exclusively on the 64-bit targets, and have previously recommended the retirement of the 32-bit targets[^1].
+
+MSYS2, a major distributor of the GNU-based Windows platform, has been [dropping some 32-bit packages](https://www.msys2.org/news/#2023-12-13-starting-to-drop-some-32-bit-packages) and [no longer distributes Clang for 32-bit](https://github.com/msys2/MINGW-packages/pull/21998), showing even their shift away from the platform.
+In response to inquiries about their opinion on reducing support for the target, [MSYS2 folks were positive](https://github.com/msys2/MINGW-packages/issues/23346).
+
+[^1]: despite saying he is "not a maintainer", mati865 is effectively our maintainer for x86_64-pc-windows-gnu and x86_64-pc-windows-gnullvm https://rust-lang.zulipchat.com/#narrow/channel/233931-t-compiler.2Fmajor-changes/topic/Demote.20.60i686-pc-windows-gnu.60.20compiler-team.23822/near/490675824
+[^2]: https://devblogs.microsoft.com/oldnewthing/20220418-00/?p=106489
+
+## Target Tier Policy Requirements
+
+With this knowledge, we can look at the [Tier 1 requirements](https://doc.rust-lang.org/1.84.1/rustc/target-tier-policy.html#tier-1-target-policy) of the target tier policy to check whether they are fulfilled.
+
+> *Tier 1 targets must have substantial, widespread interest within the developer community, and must serve the ongoing needs of multiple production users of Rust across multiple organizations or projects.*
+
+While this cannot be quantified precisely, the download counts suggest that this target is less popular than some other Tier 2 targets like FreeBSD.
+Therefore, we are going to treat this as false.
+
+> The target maintainer team must include at least 3 developers.
+
+This is not the case at all. There is currently no maintainer team.
+Though we should note that this is currently also true for many other Tier 1 targets, as this is a new rule not upheld everywhere yet.
+But experience tells that it is highly unlikely that 3 maintainers for 32 bit Windows GNU will be found.
```

Comment:
Remark: and I think it's arguably misleading to say `i686-pc-windows-gnu` is a Tier 1 target when it does not really receive Tier 1 maintenance support IMHO.